css/project.less

```diff
@@ -148,6 +152,32 @@
   &.loading .load-ui {
     display: block;
   }
+}
+
+.project-resizer-wrapper {
+  .autohide & {
+    display: none;
+  }
 
-  
-}
\ No newline at end of file
+  position: absolute;
+  height: 100%;
+  right: 0;
+  top: 0;
+
+  width: 16px;
+
+  .project-resizer {
+    &:extend(.project-resizer-wrapper);
```

Comment:
I haven't pulled this yet, so forgive me if it's a stupid question, but would it be possible to use a pseudo-element, instead of needing the handle and the wrapper both?